images/core-image-minimal-rubygems.bb

```diff
@@ -0,0 +1,21 @@
+require recipes-core/images/core-image-minimal.bb
+
+inherit testimage
+
+DESCRIPTION = "Image for testing rubygems. This automatically inherits all ruby recipes and testcases"
+LICENSE = "MIT"
+
+def rubygems_get_ruby_testcases(d):
```

Comment:
potential issue: Assignment should be 'VAR_append = " foo"' not 'VAR_append = "foo"'